GiTinder.Tests/Models/SettingsFactory.cs

```diff
@@ -0,0 +1,75 @@
+﻿using GiTinder.Models;
+
+namespace GiTinder.Tests.Models
+{
+    class SettingsFactory
+
+    {
+
+
+        public static Settings CreateSettings()
+        {
+            return new Settings();
+        }
```

Comment:
@FilipTeply factory should be used for creating a valid object
and a couple of invalid objects based on the valid's modification.
`CreateSettingsWithValidUserName ` is not going to be used anywhere, because it is a valid username, but that's not the only thing that needs to be valid

Reply:
Done.